autonomy/deploy/generators/kubernetes/templates.py

```diff
@@ -187,10 +187,12 @@
   selector:
     matchLabels:
       app: agent-node-{validator_ix}
+      valory: agent-svc
```

Comment:
Open Autonomy is not only used by Valory. This is a non-generic addition to the template. We should think of a cleaner way to do this.

Reply:
I agree 100%
I had approached it from the other direction with this pr;

https://github.com/valory-xyz/internal-infrastructure/pull/636/files#

Which allows us to deploy prior to the next release of the open-autonomy framework.